### PR TITLE
Update UML diagram

### LINKED ISSUE #100: Update uml diagrams

### PR DESCRIPTION
**Pull request by achl1012**
Closes #100

### CODE REVIEW

File: docs/diagrams/ModelClassDiagram.puml [hunk 1]
```diff
@@ -19,6 +19,7 @@ Class ModuleName
 Class Email
 Class Name
 Class Phone
+Class TelegramHandle
```
Comment (zienxu) [inline docs/diagrams/ModelClassDiagram.puml:22]: good addition

**Review by zienxu — DISMISSED** [review]
LGTM!

File: docs/diagrams/ModelClassDiagram.puml [hunk 2]
```diff
@@ -39,6 +40,7 @@ UserPrefs .up.|> ReadOnlyUserPrefs
 AddressBook *--> "1" UniquePersonList
 UniquePersonList --> "~* all" Person
 Person *--> "1" ContactType
+Person *--> "1" TelegramHandle
```
Comment (SlothyCat) [inline docs/diagrams/ModelClassDiagram.puml:43]: Good to indicate multiplicity

**Review by SlothyCat — APPROVED** [review]
LGTM! 😄

**Pull request merged**